src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -107,6 +107,7 @@ public function getConfigTreeBuilder()
                     ->beforeNormalization()->ifString()->then(function ($v) { return array($v); })->end()
                     ->prototype('scalar')->end()
                 ->end()
+                ->booleanNode('reset_services')->defaultFalse()->end()
```

Comment:
any better idea for the name?

Reply:
reset_services_on_terminate?
no more comment :)